runtime/commonTest/src/kotlinx/serialization/TuplesTest.kt

```diff
@@ -43,7 +44,7 @@ class TuplesTest : JsonTestBase() {
         val desc = PairWrapper.serializer().descriptor.getElementDescriptor(0)
         assertEquals(desc.serialName, "kotlin.Pair")
         assertEquals(
-            desc.elementDescriptors.map(SerialDescriptor::kind),
+            desc.elementDescriptors.map(kotlinx.serialization.descriptors.SerialDescriptor::kind),
```

Comment:
FQNS are not necessary

Reply:
I think this is inserted by the 'move' refactoring. Looks like an issue in it with function references